Controllers/ExampleController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Bolt\Extension\YourName\ExtensionName\Controllers;
+
+use Bolt\Application;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+class ExampleController
+{
+    /**
+     * Bolt Application instance
+     *
+     * @var Application
+     */
+    private $app;
+
+    /**
+     * Extension configuration
+     *
+     * @var array
+     */
+    private $config;
+
+    function __construct(Application $app, $config)
```

Comment:
I never wrote `public` before a constructor because a constructor is always public. But I can do this if you want.